def raise_exception():
    raise TypeError("This is a custom type error.")

print(raise_exception());